trino/auth.py

```diff
@@ -131,3 +132,38 @@ def get_exceptions(self):
 
     def handle_error(self, handle_error):
         pass
+
+
+class BearerAuth(AuthBase):
```

Comment:
Can we make this a private class in order to prevent direct use?

Reply:
Making class ‘private’ is enough or methods should also be made private? @joshthoward

Reply:
Sorry my bad, understood